### PR TITLE
Fixed faction tags in two cards' text

### PR DESCRIPTION
**Pull request by distributive**
For some reason Wyvern (the anarch draft ID) and Raman Rai (a Jinteki alliance card) use curly braces instead of square brackets when referencing their faction (e.g. `{anarch}` instead of `[anarch]`).

This causes an issue in at least two bots that use the nrdb api when rendering the affected cards and I assume this is a mistake.

### CODE REVIEW

File: pack/draft.json [hunk 1]
```diff
@@ -53,9 +53,9 @@
         "position": 7,
         "quantity": 1,
         "side_code": "runner",
-        "stripped_text": "Draft format only. You must maintain the order of your heap. Whenever you trash a Corp card, if you have more {anarch} cards installed than any other faction, shuffle the top card of your heap into your stack.",
+        "stripped_text": "Draft format only. You must maintain the order of your heap. Whenever you trash a Corp card, if you have more [anarch] cards installed than any other faction, shuffle the top card of your heap into your stack.",
```
Comment (plural) [inline pack/draft.json:56]: this one can actually just be the bare word "anarch" without the brackets.

File: pack/si.json [hunk 2]
```diff
@@ -212,9 +212,9 @@
         "position": 68,
         "quantity": 3,
         "side_code": "corp",
-        "stripped_text": "This card costs 0 influence if you have 6 or more non-alliance {jinteki} cards in your deck. Whenever you draw a card, you may lose click. If you do, reveal that card and 1 card in Archives of the same type. Swap those cards. Use this ability only once per turn.",
+        "stripped_text": "This card costs 0 influence if you have 6 or more non-alliance [jinteki] cards in your deck. Whenever you draw a card, you may lose click. If you do, reveal that card and 1 card in Archives of the same type. Swap those cards. Use this ability only once per turn.",
```
Comment (plural) [inline pack/si.json:215]: this should be the bare word "jinteki" without the brackets.

**Review by plural — COMMENTED** [review]
thanks!

**Comment by distributive** [comment]
The issues should be fixed

**Pull request merged**